core/utils/file_utils.py

```diff
@@ -50,6 +50,20 @@ def clean(folder):
     def exists(folder):
         return os.path.isdir(folder)
 
+    @staticmethod
+    def get_current_folder():
```

Comment:
I think we do not need this and `navigate_to`.
We have `cwd` param we can pass to `run()` and make it run the things in specific folder.

Reply:
👍